sources/packages/web/src/views/institution/locations/programs/LocationProgramOffering.vue

```diff
@@ -1,42 +1,58 @@
 <template>
-  <v-container>
-    <header-navigator
-      title="Program detail"
-      :routeLocation="getRouteLocation()"
-      :subTitle="subTitle"
-    >
-      <template #buttons>
-        <v-btn
-          v-if="showActionButtons"
-          color="primary"
-          variant="outlined"
-          @click="assessOffering(OfferingStatus.Declined)"
-          >Decline</v-btn
-        >
-        <v-btn
-          class="ml-2"
-          color="primary"
-          v-if="showActionButtons"
-          @click="assessOffering(OfferingStatus.Approved)"
-          >Approve offering</v-btn
-        >
-      </template></header-navigator
-    >
-    <program-offering-detail-header
-      v-if="offeringId"
-      class="m-4"
-      :headerDetails="{
-        ...initialData,
-        status: initialData.offeringStatus,
-        institutionId: institutionId,
-      }"
-    />
+  <full-page-container>
+    <template #header>
+      <header-navigator
+        title="Program detail"
+        :routeLocation="getRouteLocation()"
```

Comment:
Nothing major, but is it possible to change this to :routeLocation="getRouteLocation" ?